HW2/css/styles.css

```diff
@@ -1,13 +1,11 @@
 .container {
   width: 1280px;
-  height: 1282px;
   margin: auto;
+  font-family: sans-serif;
 }
 .header {
```

Comment:
Plz try to remove unused properties
![image](https://user-images.githubusercontent.com/44003977/217986764-0ad151da-07a1-48a0-a68a-3b634f816c89.png)